### PR TITLE
Reserve your pricing

### PR DESCRIPTION
**Pull request by chexton**

### CODE REVIEW

**Comment by probablyrory** [comment]
I've added a link to this page in the current annual pricing help doc. Other than that, i'm not vibing on the title but I can't think of anything better using the term 'reserve'.

**Comment by probablyrory** [comment]
Actually the only other thing would be having a link to the pricing page... We have annual pricing on that page so will it line up 20% doesn't equal 2 months off right?

**Comment by dyspop** [comment]
"Lock in"

**Comment by chexton** [comment]
@dyspop thanks for the feedback! I took a second look at this and feel the terminology is better. It's much more in the spirit of what we're trying to achieve here – a positive outcome for both parties. Plus, I think the language is easier to understand and simplicity/transparency are always a winner.

**Comment by chexton** [comment]
FYI @probablyrory I made some updates to this.

**Comment by probablyrory** [comment]
👍  Yep, reads better now. :shipit:

**Comment by miquille** [comment]
I feel like the changing plan thing still feels somewhat hard to follow. I think more examples would be good.

I think something like, Buy a 2 year contract, originally at X per month, then after 1 year you're now at plan Y. 

Then a table that shows

|Plan         |  After 1 Year  | After 2 years |
+---------+-------------+--------------+
|Original    | $x * 12          | $x * 12 + $y * 12 |
|Reserved | Total (but you really only paid ($x * 12) * .8) | Total + overcharge |

And hopefully that shows the savings as still worth it? Feel free to make that 1 year / 6 month interval if the chart makes it seem like it's better to buy 1 year and be flexible.

**Comment by chexton** [comment]
🚢  away!
